pavelib/assets.py

```diff
@@ -49,6 +49,7 @@
     'underscore.string/dist/underscore.string.js',
     'picturefill/dist/picturefill.js',
     'backbone/backbone.js',
+    'backbone.paginator/lib/backbone.paginator.js',
```

Comment:
As mentioned before. I'd copy this from the UI Toolkit's node_modules directory.